varnish/metadata.csv

```diff
@@ -322,22 +424,47 @@ varnish.SMA.Transient.c_req,rate,,event,,Times the storage has been asked to pro
 varnish.SMA.Transient.g_alloc,gauge,,event,,Storage allocations outstanding.,0,varnish,SMA Transient g alloc
 varnish.SMA.Transient.g_bytes,gauge,,byte,,Space allocated from the storage.,0,varnish,SMA Transient g bytes
 varnish.SMA.Transient.g_space,gauge,,byte,,Space left in the storage.,0,varnish,SMA Transient g space
-varnish.sms_balloc,rate,,byte,,SMS space allocated.,0,varnish,sms balloc
-varnish.sms_bfree,rate,,byte,,SMS space freed.,0,varnish,sms bfree
-varnish.sms_nbytes,rate,,byte,,SMS outstanding space.,0,varnish,sms nbytes
-varnish.sms_nobj,rate,,event,,SMS outstanding allocations.,0,varnish,sms nobj
-varnish.sms_nreq,rate,,event,,SMS allocator requests.,0,varnish,sms nreq
+varnish.sms_balloc,gauge,,byte,,SMS space allocated.,0,varnish,sms balloc
+varnish.sms_bfree,gauge,,byte,,SMS space freed.,0,varnish,sms bfree
+varnish.sms_nbytes,gauge,,byte,,SMS outstanding space.,0,varnish,sms nbytes
+varnish.sms_nobj,gauge,,event,,SMS outstanding allocations.,0,varnish,sms nobj
+varnish.sms_nreq,gauge,,event,,SMS allocator requests.,0,varnish,sms nreq
```

Comment:
This is a mistake introduced in https://github.com/DataDog/integrations-core/pull/8060
These metrics are not actually emitted in CI